src/material/autocomplete/autocomplete.html

```diff
@@ -1,5 +1,5 @@
 <ng-template>
-  <div class="mat-autocomplete-panel" role="listbox" [id]="id" [ngClass]="_classList" #panel>
+  <div class="mat-autocomplete-panel" role="listbox" aria-label="Search suggestions" [id]="id" [ngClass]="_classList" #panel>
```

Comment:
I don't think that we can do this with a static string, because it won't work for apps in non-English languages. The way we did it for `mat-select` (which has an identical setup) was to either forward the `aria-label` from the trigger element, or to set `aria-labelledby` to the label of the trigger's form field. Also note that if we go down this route, we have to make sure that we don't have both `aria-label` and `aria-labelledby` at the same time, because they're mutually exclusive.